packages/sqlite/src/index.ts

```diff
@@ -12,34 +14,6 @@ const sqliteFormat: FormatConfig = {
   formatValue: (value) => ({placeholder: '?', value}),
 };
 
-export enum DatabaseConnectionMode {
-  ReadOnly = sqlite.OPEN_READONLY,
-  ReadWrite = sqlite.OPEN_READWRITE,
-  // tslint:disable-next-line:no-bitwise
-  ReadWriteCreate = sqlite.OPEN_READWRITE | sqlite.OPEN_CREATE,
-  // tslint:disable-next-line:no-bitwise
-  ReadCreate = sqlite.OPEN_READONLY | sqlite.OPEN_CREATE,
-  Create = sqlite.OPEN_CREATE,
-}
```

Comment:
Does the new driver still support specifying the `mode`?

Reply:
It provides different options for this: https://github.com/WiseLibs/better-sqlite3/blob/master/docs/api.md#new-databasepath-options, as you can see it has `readonly` and `fileMustExists` that roughly map to the above combination. I thought it was better to expose those directly given this is a breaking change anyway.